docs/plugin-development.rst

```diff
@@ -199,6 +199,44 @@ could access it using ::
 
 Where the second argument to ``model`` is the name of your plugin.
 
+Adding a custom permission flag
```

Comment:
These are great docs.  Please add a discussion of the fact that a flag can be `admin` or not, and if there are any other permission levels for flags.